tests/stub/basic_query/test_basic_query.py

```diff
@@ -316,35 +337,65 @@ def test_5x0_populates_path_element_ids_with_only_string(self):
-
-        self.assertIsInstance(result.values[0], CypherPath)
-        path = result.values[0]
-        self.assertIsInstance(path.nodes, CypherList)
-        self.assertIsInstance(path.relationships, CypherList)
-        nodes = path.nodes.value
-        rels = path.relationships.value
-        # node ids
-        self._assert_is_unset_id(nodes[0].id)
-        self.assertEqual(CypherString("n1-1"), nodes[0].elementId)
-        # rel ids
-        self._assert_is_unset_id(rels[0].id)
-        self.assertEqual(CypherString("r1-2"), rels[0].elementId)
-        # rel start/end ids
-        self._assert_is_unset_id(rels[0].startNodeId)
-        self._assert_is_unset_id(rels[0].endNodeId)
-        self.assertEqual(CypherString("n1-1"), rels[0].startNodeElementId)
-        self.assertEqual(CypherString("n1-3"), rels[0].endNodeElementId)
-
-        self._session.close()
-        self._session = None
-        self._server.done()
+        with self._get_session("single_result.script",
+                               script_params) as session:
+            result_handle = session.run("MATCH p = ()--()--() "
+                                        "RETURN p LIMIT 1")
+
+            result = result_handle.next()
+
+            self.assertIsInstance(result.values[0], CypherPath)
+            path = result.values[0]
+            self.assertIsInstance(path.nodes, CypherList)
+            self.assertIsInstance(path.relationships, CypherList)
+            nodes = path.nodes.value
+            rels = path.relationships.value
+            # node ids
+            self._assert_is_unset_id(nodes[0].id)
+            self.assertEqual(CypherString("n1-1"), nodes[0].elementId)
+            # rel ids
+            self._assert_is_unset_id(rels[0].id)
+            self.assertEqual(CypherString("r1-2"), rels[0].elementId)
+            # rel start/end ids
+            self._assert_is_unset_id(rels[0].startNodeId)
+            self._assert_is_unset_id(rels[0].endNodeId)
+            self.assertEqual(CypherString("n1-1"), rels[0].startNodeElementId)
+            self.assertEqual(CypherString("n1-3"), rels[0].endNodeElementId)
+
+            self._server.done()
+
+    @driver_feature(types.Feature.BOLT_5_0,
+                    types.Feature.DETAIL_THROW_ON_MISSING_ID)
+    def test_5x0_throws_on_reading_ids(self):
+        script_params = {
+            "#BOLT_PROTOCOL#": "5.0",
+            "#RESULT#":
+                '{"..": ['
+                '{"()": [null, ["l"], {}, "n1-1"]}, '
+                '{"->": [null, null, "RELATES_TO", null, {}, '
+                '"r1-2", "n1-1", "n1-3"]}, '
+                '{"()": [null, ["l"], {}, "n1-3"]}, '
+                '{"->": [null, null, "RELATES_TO", null, '
+                '{}, "r1-4", "n1-3", "n1-1"]}, '
+                '{"()": [null, ["l"], {}, "n1-1"]}'
+                ']}'  # noqa: Q000
+        }
+        cases = [
+            "n.0.id",
+            "r.0.id",
+            "r.0.startnodeid",
```

Comment:
I'm wondering if they should be in line with what the attributes of `types.CypherRelationship` are.